.github/workflows/release.yml

```
@@ -50,166 +71,114 @@ jobs:
     steps:
       - name: "[DEBUG] Print Variables"
         run: |
-          echo The last commit sha in the release:  ${{ inputs.sha }}
-          echo The release version number:          ${{ inputs.version_number }}
-          echo The path to the changelog markdpown: ${{ inputs.changelog_path }}
-          echo This is a test run:                  ${{ inputs.test_run }}
-          echo Python target version:               ${{ env.PYTHON_TARGET_VERSION }}
-          echo Artifact retention days:             ${{ env.ARTIFACT_RETENTION_DAYS }}
-
-  unit:
-    name: Unit Test
-    runs-on: ubuntu-latest
+          echo The last commit sha in the release: ${{ inputs.sha }}
+          echo The branch to release from:         ${{ inputs.target_branch }}
+          echo The release version number:         ${{ inputs.version_number }}
+          echo Build script path:                  ${{ inputs.build_script_path }}
+          echo Environment setup script path:      ${{ inputs.env_setup_script_path }}
+          echo AWS S3 bucket name:                 ${{ inputs.s3_bucket_name }}
+          echo Package test command:               ${{ inputs.package_test_command }}
+          echo Test run:                           ${{ inputs.test_run }}
+
+  # The Spark repository uses CircleCi to run integration tests.
```

Nit: "CircleCI" rather than "CircleCi".

```suggestion
  # The Spark repository uses CircleCI to run integration tests.
```

---

Fixed in https://github.com/dbt-labs/dbt-spark/pull/592/commits/517bdbf6f67d99e30b71e8c2f7fe44a849f0eaf2